.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: objective-c
 osx_image: xcode7.1
 
+before_install:
+  - brew update
+  - brew install swiftlint 
```

Comment:
I read the Travis doc, and it seems that install should be used to install the project not the tools.
Like installing dependencies.